iroha/src/sumeragi.rs

```diff
@@ -1071,19 +1062,27 @@ mod tests {
         let peers = vec![
             PeerId {
                 address: "127.0.0.1:7878".to_string(),
-                public_key: [1u8; 32],
+                public_key: KeyPair::generate()
+                    .expect("Failed to generate KeyPair.")
+                    .public_key,
             },
             PeerId {
                 address: "127.0.0.1:7879".to_string(),
-                public_key: [2u8; 32],
+                public_key: KeyPair::generate()
+                    .expect("Failed to generate KeyPair.")
+                    .public_key,
             },
             PeerId {
                 address: "127.0.0.1:7880".to_string(),
-                public_key: [3u8; 32],
+                public_key: KeyPair::generate()
+                    .expect("Failed to generate KeyPair.")
+                    .public_key,
             },
             PeerId {
                 address: "127.0.0.1:7881".to_string(),
-                public_key: [4u8; 32],
+                public_key: KeyPair::generate()
+                    .expect("Failed to generate KeyPair.")
```

Comment:
So many `expect`s... I started to think that we need a separate function for such cases likes tests :)
Actually, it returns `Result`, so maybe it'll be fine to just `unwrap`

Reply:
We do not use `unwrap` - I will add this to our `CONTRIBUTING.md`